script/setup_docker_prereqs

```diff
@@ -1,63 +1,120 @@
 #!/bin/bash
 # Install requirements and build dependencies for Home Assinstant in Docker.
 
+INSTALL_TELLSTICK="${INSTALL_TELLSTICK:-yes}"
+INSTALL_OPENALPR="${INSTALL_OPENALPR:-yes}"
+INSTALL_FFMPEG="${INSTALL_FFMPEG:-yes}"
+INSTALL_PHANTOMJS="${INSTALL_PHANTOMJS:-yes}"
+
 # Required debian packages for running hass or components
 PACKAGES=(
+  # build-essential is required for python pillow module on non-x86_64 arch
+  build-essential
```

Comment:
build-essential is needed to install home-assistant on non-x86_64 platforms.

You can see how pillow determines library paths (here)[https://github.com/python-pillow/Pillow/blob/8b2681e369af08843aef085bbb061aa8929ec0eb/setup.py#L197-L226].